index.html

```diff
@@ -1057,6 +1057,37 @@ <h2>The <dfn>InteractionOutput</dfn> interface</h2>
     <section><h3>The <dfn>check data schema</dfn> algorithm</h3>
       To run the <a>check data schema</a> steps on |payload| and |schema:object|,
       <ol>
+        <li>
+          If |schema| is `null` or `undefined`, return `undefined`.
+        </li>
+        <li>
+          <!--
+            TODO: Should we just continue with the rest of the algorithm here?
+          -->
+          If |schema|.|default| is not |undefined| and |payload| is undefined,
+          let |payload| be the value of |schema|.|default|.
```

Comment:
I now had another look into the JSON Schema Draft 7 specification (which is the basis for the `DataSchema` vocabulary in the TD) and noticed that `default` is actually only an "Annotation" (see [here](https://json-schema.org/draft-07/draft-handrews-json-schema-validation-01#annotations) and [here](https://json-schema.org/draft-07/draft-handrews-json-schema-validation-01#rfc.section.10)) that is supposed to be a hint used in documentation but not for validation.

On the one hand, I think that should be clarified in the TD document (as it is only implied by the reference to JSON Schema, which is only informative, though). On the other hand, I think that means we can simply remove this step from the algorithm as the default value is not supposed to be provided by the consumer if no payload is given.

Reply:
I now removed the step for `default` in b38cf776c5bd41be7bdc81f2ed4ea6701934160a.